client-js/app/scripts/echo-server.ts

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
any particular reason for all the extra blanks lines here and in registry.ts and rpc.ts?